ext/tests/name_uncastable_object.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+OpenCensus Trace: Providing integer as name
+--FILE--
+<?php
+
+class CastableObject
```

Comment:
`CastableObject`

Is the name intentional?

Reply:
Ah, I'll rename to `UncastableObject` it was duplicated from the `name_castable_object.phpt` test